cob_navigation_config/CMakeLists.txt

```diff
@@ -5,19 +5,30 @@ find_package(catkin REQUIRED)
 
 catkin_package()
 
+set(ROBOTLIST
+  cob3-1
+  cob3-2
+  cob3-3
+  cob3-4
+  cob3-5
+  cob3-6
+  cob4-1
+  cob4-2
+  desire
+  raw3-1
+  raw3-2
+  raw3-3
+  raw3-4
+  raw3-5
+  raw3-6
+)
```

Comment:
So should I remove the unsupported ones?
I thought I'll just keep them in for now, as there are also several currently supported ones still missing...